proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -112,3 +128,45 @@ message QueryValidatorProviderAddrResponse {
   // The address of the validator on the provider chain
   string provider_address = 1;
 }
+
+message QueryPendingSlashPacketsRequest {}
+
+message QueryPendingSlashPacketsResponse {
+  // current slash_meter state
+  int64 slash_meter = 1;
+  // allowance of voting power units (int) that the slash meter is given per replenish period
+  // this also serves as the max value for the meter.
+  int64 slash_meter_allowance = 2;
+  google.protobuf.Timestamp last_replenish = 3
+    [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+  google.protobuf.Timestamp next_replenish = 4
+    [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+  repeated PendingSlashPacket packets = 5;
+}
+
+message QueryPendingConsumerPacketsRequest {
+  string chain_id = 1;
+}
+
+message QueryPendingConsumerPacketsResponse {
+  string chain_id = 1;
+  uint64 size = 2;
+  repeated PendingPacketWrapper packets = 3
+    [(gogoproto.nullable) = false];
+}
+
+message PendingSlashPacket {
```

Comment:
Do we strictly need this wrappers and such? Or can we consolidate with existing types? 

We can talk about this next week

Reply:
You can't store dissimilar types in a protobuf array. Wrapping solves that issue. It's not pretty, I agree.

Reply:
Makes sense, `PendingPacketWrapper` is needed. I've removed `PendingSlashPacket` type since it is essentially the `GlobalSlashEntry` type. See #613

Reply:
Got it, I understand now. 

Note I've cherry-picked #613 into this PR, and adjusted one of the wrapper types in https://github.com/cosmos/interchain-security/commit/3fabcd8c23c72987dd0916b56a97cba11abe59ad.

This will make it so that we more composability of types and no fields that are covered by an already existing type